.github/workflows/test.yaml

```diff
@@ -3,14 +3,16 @@ name: Run tests
 on:
   push:
     branches: [ trunk ]
+    tags:
+      - twisted-*
   pull_request:
     branches: [ trunk ]
 
 jobs:
   testing:
     runs-on: ubuntu-20.04
     env:
-      TOXENV: "alldeps-withcov-posix,coverage-prepare,codecov-push,coveralls-push"
+      TOXENV: "alldeps-withcov-posix,coverage-prepare,codecov-push,coveralls-push,release"
```

Comment:
test.yaml is a very generic name...

and this is only testing on Ubuntu 20.04 .... we don't test on Windows and macOS

Do we need to run the tests here?

I was thinking that we can have a simple wheel + sdist publishing workflow which will run a single job which will call ciwheelbuilder and then publish

Reply:
>  Windows and macOS

I think the plan is that we're going to run this workflow on MacOS and Windows soon, and remove azure pipelines

Reply:
OK... but do we need to create the sdist and test using twine for each job from the matrix?

Can we move `tox -e release` as a step in the `pypi-publish` job?

There are a lot of tox environments executed in the matrix...if possible, I would prefer to have all the wheel/sdist related task in the dedicated pypi publish job. Does it make sense?